src/python/pants/backend/python/tasks/build_local_python_distributions.py

```diff
@@ -39,53 +43,73 @@ def product_types(cls):
   def prepare(cls, options, round_manager):
     round_manager.require_data(PythonInterpreter)
 
+  @classmethod
+  def implementation_version(cls):
+    return super(BuildLocalPythonDistributions, cls).implementation_version() + [('BuildLocalPythonDistributions', 1)]
+
+  @classmethod
+  def subsystem_dependencies(cls):
+    return super(BuildLocalPythonDistributions, cls).subsystem_dependencies() + (PythonDistBuildEnvironment.scoped(cls),)
+
+  # TODO: seriously consider subclassing UnixCCompiler as well as the build_ext
```

Comment:
This is the second reference to this strategy. Probably justifies a github issue to link to in both places.

Reply:
I will make an issue describing this -- I've currently just concatenated the comments but will make an issue and just link to that in a comment.